code_of_conduct.md

```diff
@@ -0,0 +1,129 @@
+
```

Comment:
I would recommend to have this and the README on other commit. A simple reset and re-commit will do the trick.